src/metrics/peerstore_test.go

```diff
@@ -33,3 +34,31 @@ func Test_PeerStore(t *testing.T) {
 	require.Equal(t, ok, true)
 	require.Equal(t, p.ClientName, "Client3")
 }
+
+func TestDb(t *testing.T){
+	peerStore := NewPeerStore()
+	peerStore.AddPeer(Peer{
+		PeerId: "Peer1",
+		ClientName: "Client1",
+	})
+	peerStore.AddPeer(Peer{
+		PeerId: "Peer2",
+		ClientName: "Client2",
+	})
+
+	err := peerStore.PersistToDb("../../test.db")
```

Comment:
It's just arbitrary, I thought that if an error caused the test to stop and not delete the file, having the test.db in the root directory would be easier to locate and delete instead of being embedded between the source files